### PR TITLE
Tensorflow Lite demo app for Android: add support for floating point models as Inception-v3

### PR DESCRIPTION
**Pull request by Johnson145**
Although the new Lite interface does support float models as well, the current Android demo app does only support quantized models. Furthermore, it isn't obvious to transfer the code from the quantized version to the floating point model. [Based on this discussion](https://github.com/tensorflow/tensorflow/issues/14719) I integrated the [Inception-v3 slim model](https://storage.googleapis.com/download.tensorflow.org/models/tflite/inception_v3_slim_2016_android_2017_11_10.zip) as an alternative to the existing MobileNet.

_Remaining TODO_:
The confidence scores returned by the inception net are not in [0,1] yet. Besides that, the inference itself seems to work. So the correct results are listed on top, but the confidence score isn't normalized. Maybe the given model doesn't include a Softmax layer and ends with the logits? I'm not sure about this. Any help is appreciated.

### CODE REVIEW

**Comment by Johnson145** [comment]
CLA problem should be solved.

**Comment by freedomtan** [comment]
For the output Inception V3 slim 2016 tflite mode, see https://github.com/tensorflow/tensorflow/issues/16851

**Review by andrehentz — APPROVED** [review]
